compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/types/BTupleType.java

```diff
@@ -35,39 +36,40 @@
  */
 public class BTupleType extends BType implements TupleType {
 
-    public List<BType> tupleTypes;
+    public List<BTupleMember> memberTypes;
     public BType restType;
     public Boolean isAnyData = null;
     public boolean resolvingToString = false;
     public boolean isCyclic = false;
 
     private BIntersectionType intersectionType = null;
 
-    public BTupleType(List<BType> tupleTypes) {
+    public BTupleType(List<BTupleMember> memberTypes) {
         super(TypeTags.TUPLE, null);
-        this.tupleTypes = tupleTypes;
+        this.memberTypes = memberTypes;
     }
 
-    public BTupleType(BTypeSymbol tsymbol, List<BType> tupleTypes) {
+    public BTupleType(BTypeSymbol tsymbol, List<BTupleMember> memberTypes) {
         super(TypeTags.TUPLE, tsymbol);
-        this.tupleTypes = tupleTypes;
+        this.memberTypes = memberTypes;
     }
 
-    public BTupleType(BTypeSymbol tsymbol, List<BType> tupleTypes, boolean isCyclic) {
+    public BTupleType(BTypeSymbol tsymbol, List<BTupleMember> memberTypes, boolean isCyclic) {
         super(TypeTags.TUPLE, tsymbol);
-        this.tupleTypes = tupleTypes;
+        this.memberTypes = memberTypes;
         this.isCyclic = isCyclic;
     }
 
-    public BTupleType(BTypeSymbol tsymbol, List<BType> tupleTypes, BType restType, long flags) {
+    public BTupleType(BTypeSymbol tsymbol, List<BTupleMember> memberTypes, BType restType, long flags) {
         super(TypeTags.TUPLE, tsymbol, flags);
-        this.tupleTypes = tupleTypes;
+        this.memberTypes = memberTypes;
         this.restType = restType;
     }
 
-    public BTupleType(BTypeSymbol tsymbol, List<BType> tupleTypes, BType restType, long flags, boolean isCyclic) {
+    public BTupleType(BTypeSymbol tsymbol, List<BTupleMember> memberTypes, BType restType, long flags,
+                      boolean isCyclic) {
         super(TypeTags.TUPLE, tsymbol, flags);
-        this.tupleTypes = tupleTypes;
+        this.memberTypes = memberTypes;
```

Comment:
Please fix 
- members - for `BTupleMember`s
- memberTypes - for the type of the members